homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -248,6 +250,10 @@ def device_state_attributes(self):
                 ATTR_STATUS: str(self.vacuum_state.state)
                 })
 
+            if self.last_clean:
```

Comment:
`self.last_clean` not defined yet.

Reply:
Thanks for the review, it is now initialized in `__init__`.